### PR TITLE
Release 0.3.0

### LINKED ISSUE #107: Prepare for release 0.3.0
## Depends on tasks:
- #3 

## Description:
Prepare the documentation and source for release.

### User requirements:
User should be able to go to the release page in repo, get the information about changes that were done and be able to download/setup the app

### Dev requirements and restrictions:

- Create a new branch for the release with the name "release-0.3.0" and set it up for release;
- Create a new branch to bump the version, if requires.
- Check the CHANGELOG.md, verify that there are no missed changes, and update with release information;
- Add the necessary info to the [Release_notes](https://github.com/ios-course/ironfoudation-team-project/blob/main/release_notes.md) in a user-friendly format based on CHANGELOG.md.
- Before merging to the main branch set up the tag "v.0.3.0" on the last commit before the merge commit.
- Publish the Release under the Release [section on GitHub.](https://github.com/ios-course/ironfoudation-team-project/releases)

### PR DESCRIPTION
**Pull request by julyakmita**
## Summary
This PR is a part of the work to resolve #107 

## Changes
- Update `Changelog.md` with the info related to release 0.3.0
- Update `Release_notes.md` with the info related to release 0.3.0

## Checks to complete
- [x] My code follows our style guide.
- [x] I've performed a self-review of my code.
- [x] PR's title has a conventional commit style (short and descriptive).
- [x] PR has assignee, reviewers, milestone, it is properly linked to the project and to the issue.

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -6,6 +6,16 @@ The file describes all the changes, that are affected the usage of the applicati
...
+## 0.3.0
+
+### New features
+
+`Log in`: Add possibility for user to select how to run the app - as a pet owner or a pet sitter.
+
+`Logout`: The user are able to change the selected role (owner or sitter). The  Logout button is displayed in the navigation bar.
+
+`Copy info`: Users are able to copy the name and the phone number of a pet sitter by a long tapping on the sitter's full name and phone information.
+
```
Comment (marinkman) [inline release_notes.md:18]: I think you need to add another empty line here to be consistent.

File: release_notes.md [hunk 2]
```diff
@@ -6,6 +6,16 @@ The file describes all the changes, that are affected the usage of the applicati
 
 The format is based on [Release_notes format](https://github.com/ios-course/ironfoudation-team-project/wiki/Release_notes-format).
 
+## 0.3.0
+
+### New features
+
+`Log in`: Add possibility for user to select how to run the app - as a pet owner or a pet sitter.
```
Comment (marinkman) [inline release_notes.md:13]:
```suggestion
`Log in`: Add the possibility for the user to select how to run the app - as a pet owner or a pet sitter.
```

**Review by marinkman — COMMENTED** [review]
Consider improvements:

File: release_notes.md [hunk 3]
```diff
@@ -6,6 +6,16 @@ The file describes all the changes, that are affected the usage of the applicati
...
+
+`Logout`: The user are able to change the selected role (owner or sitter). The  Logout button is displayed in the navigation bar.
```
Comment (marinkman) [inline release_notes.md:15]:
```suggestion
`Logout`: The user are able to change the selected role (owner or sitter). The Logout button is displayed in the navigation bar.
```

File: release_notes.md (hunk 1 above)
Comment (marinkman) [reply release_notes.md:18]: There are two blank lines between the first and second release.

File: release_notes.md (hunk 2 above)
Comment (julyakmita) [reply release_notes.md:13]: Thanks. Fixed it.

File: release_notes.md (hunk 1 above)
Comment (julyakmita) [reply release_notes.md:18]: Fixed it. Thanks.

**Review by marinkman — APPROVED** [review]
LGTM

**Review by kirpichonok — APPROVED** [review]
LGTM

**Comment by julyakmita** [comment]
This PR was closed, because the release branch should be done from another commit.

**Pull request closed without merging**